### PR TITLE
Susd peg

### PR DESCRIPTION
**Pull request by dvd-schwrtz**

### CODE REVIEW

File: src/config/helpers.js [hunk 1]
```diff
@@ -156,50 +156,29 @@ export function* getUniswapV2SethPrice(snxjs) {
 	);
 }
 
+export function* getCurveLatestSwapPrice(snxjs) {
+	const curveContract = new snxjs.ethers.Contract(
```
Comment (evgenyboxer) [inline src/config/helpers.js:160]: hmm... this should probably be only instantiated once (out of the function)

File: src/actions/markets.js [hunk 2]
```diff
@@ -2,12 +2,11 @@ import {
...
 	FETCH_SNX_CURRENCY_PRICE,
 	FETCH_SETH_CURRENCY_PRICE,
+	FETCH_SUSD_CURRENCY_PRICE,
 	FETCH_SNX_CURRENCY_SUCCESS,
-	FETCH_NUSD_CURRENCY,
-	FETCH_NUSD_CURRENCY_SUCCESS,
 } from './actionTypes';
 
-export const fetchSNX = () => ({
+export const fetchSnx = () => ({
```
Comment (evgenyboxer) [inline src/actions/markets.js:9]: shouldn't we keep it as `fetchSNX`?

**Comment by evgenyboxer** [comment]
@dvd-schwrtz mostly lgtm, the one that needs to be addressed is the contract creation happening on every function call.

File: src/actions/markets.js (hunk 2 above)
Comment (dvd-schwrtz) [reply src/actions/markets.js:9]: No, I like this better. The whole codebase has shitty naming conventions I will change over time. it is just a bit of a mess with the naming conventions

File: src/config/helpers.js (hunk 1 above)
Comment (dvd-schwrtz) [reply src/config/helpers.js:160]: it only needs to be called once. We don't call the method again, so it is fine to do it here

**Comment by dvd-schwrtz** [comment]
> @dvd-schwrtz mostly lgtm, the one that needs to be addressed is the contract creation happening on every function call.

This is not an issue since the method is only called at mount of the page so we should be good.

**Comment by dvd-schwrtz** [comment]
> what changed in package-lock.json?

Honestly not sure, I rebased on master. Everything seems to be working fine. I think its ok to merge. I did install and uninstall ethers at one point when I was trying to fix a problem with the abi format, so that could have caused the change.

File: src/actions/markets.js (hunk 2 above)
Comment (evgenyboxer) [reply src/actions/markets.js:9]: haha ok.. well you know this repo better.

**Pull request merged**